### PR TITLE
IP-1211: Log When Job Finishes

### PR DESCRIPTION
**Pull request by evaninja**
**JIRA**: https://clever.atlassian.net/browse/IP-1211

**Overview**: Uses log-routing to log when a job finishes to SignalFx, along with the payload and pass/fail. Also imports kayvee v6 and updates our usage of Errorf to meet new linting standards.

**Testing**: Wrote unit test for new logger package + ran locally.

**Roll Out**:
- [ ] Remember to deploy both `s3-to-redshift` and `s3-to-redshift-fast`

CC @xavi-

### CODE REVIEW

File: logger/logger.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
+package logger
+
+import (
+	"gopkg.in/Clever/kayvee-go.v6/logger"
+)
+
+var log logger.KayveeLogger
```
Comment (bstein-clever) [inline logger/logger.go]: I really wish there was a way so we didn't have to do this boilerplate.

**Comment by bstein-clever** [comment]
A couple of nits, otherwise LGTM

File: kvconfig.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+routes:
+  job-finished:
+    matchers:
+      title: [ "job-finished" ]
+    output:
+      type: "alerts"
+      series: "job.finished"
+      dimensions: [ "function", "payload" ]
+      value: "value"
+      stat_type: "gauge"
```
Comment (xavi-) [inline kvconfig.yml]: I'm assuming that "value" has a the delay value?

File: logger/logger.go (hunk 1 above)
Comment (xavi-) [inline logger/logger.go]: Creating your new logger shouldn't be required.  It should be okay to use `logger.KayveeLogger` directly.

File: kvconfig.yml (hunk 2 above)
Comment (evaninja) [inline kvconfig.yml]: We log a value of 1 if the job completed successfully, 0 otherwise

Comment (xavi-) [inline kvconfig.yml]: Making it a `counter` might make more sense

File: main.go [hunk 3]
```diff
@@ -6,15 +6,18 @@ import (
...
+	"path"
 	"strings"
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/kardianos/osext"
 	env "github.com/segmentio/go-env"
+	kvlogger "gopkg.in/Clever/kayvee-go.v6/logger"
```
Comment (evaninja) [inline main.go]: 👍

File: kvconfig.yml (hunk 2 above)
Comment (evaninja) [inline kvconfig.yml]: Talked with Xavi offline - we are shooting for a gauge

**Pull request merged**